adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/BrokerProxy.cs

```diff
@@ -364,17 +364,13 @@ public Intent GetIntentForBrokerActivity(AuthenticationRequest request, Activity
                     intent.PutExtra(BrokerConstants.BrokerRequest, BrokerConstants.BrokerRequest);
                 }
             }
-            catch (OperationCanceledException e)
-            {
-                RequestContext.Logger.Error(e);
-                RequestContext.Logger.ErrorPii(e);
-            }
             catch (Exception e)
             {
                 // Authenticator gets problem from webrequest or file read/write
                 var ex = new AdalException("Authenticator cancels the request", e);
```

Comment:
>"Authenticator cancels the request" [](start = 43, length = 35)

imo, the exeptions are missing an id, or somehow it should be easy for the developer to do the right thing in case an exception occurs. Is this message available in a const anywhere?
should it be cancelled instead?

Reply:
it's coming from the broker intent, so it would make sense to make it a const message, but leave as is. our cancelled one is for user cancelling auth, which is not the case here.

Reply:
done.

Reply:
still not sure it should be valle cancells

---
In reply to: [205926754](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1140#discussion_r205926754) [](ancestors = 205926754)